taskcluster/decoder-build.sh

```diff
@@ -4,10 +4,7 @@ set -xe
 
 source $(dirname "$0")/../tc-tests-utils.sh
 
-source ${DS_ROOT_TASK}/DeepSpeech/tf/tc-vars.sh
+export OS=$(uname)
```

Comment:
There's already this in `tc-tests-utils.sh`

Reply:
Maybe it's the lack of `export` there, but without this the files downstream were picking up the `export OS` in tf-vars.sh, not `tc-tests-utils.sh`. In any case, I'll add the `export` in `tc-tests-utils.sh` instead.